zenoh-ext/examples/Cargo.toml

```diff
@@ -34,7 +34,7 @@ default = []
 [dependencies]
 tokio = { workspace = true, features = ["rt", "sync", "time", "macros", "io-std"] }
 futures = { workspace = true }
-zenoh = { workspace = true, features = ["unstable"], default-features = false }
+zenoh = { workspace = true, features = ["unstable", "internal_config"], default-features = false }
```

Comment:
Why there is the need of enabling `internal_config`?

Reply:
zenoh-ext uses direct access to config fields. Before it was hidden as the "unstable" included "internal_config"
Later I think we need to remove this feature https://github.com/eclipse-zenoh/zenoh/issues/1655